controllers/auth.js

```diff
@@ -77,12 +151,27 @@ const githubAuthCallback = (req, res, next) => {
       userData = {
         github_id: user.username,
         github_display_name: user.displayName,
+        email: user._json.email,
         github_created_at: Number(new Date(user._json.created_at).getTime()),
         github_user_id: user.id,
         created_at: Date.now(),
         updated_at: null,
       };
 
+      if (userData.email === null) {
```

Comment:
- this will also check for undefined along with null. 
```suggestion
      if (!userData.email) {
```

Reply:
I will update it.